app/build.gradle

```diff
@@ -251,7 +251,6 @@ android {
             }
             buildConfigField "String", "HVR_ML_API_KEY", "\"${getHVRMLApiKey()}\""
             buildConfigField "String", "MK_API_KEY", "\"\""
-            buildConfigField "Boolean", "WEBVIEW_IN_PHONE_UI", "true"
```

Comment:
I suggest to keep this build parameter and use something like this to change its value:

```js
    applicationVariants.all { variant ->
        def platform = variant.productFlavors.get(0).name
        def country = variant.productFlavors.get(2).name

        if (platform.toLowerCase().startsWith('hvr') && country == 'cn')
            variant.buildConfigField "Boolean", "WEBVIEW_IN_PHONE_UI", "false"
    }
```

Reply:
OK